### PR TITLE
Elevation review 3 items

### PR DESCRIPTION
**Pull request by Leehman**
Please review the elevation examples. In addition layout/elevation.js will have to modified to remove 'playgound'

### CODE REVIEW

**Comment by Leehman** [comment]
snap 2018-01-29 at 10 46 49
snap 2018-01-29 at 10 47 19

**Comment by Leehman** [comment]
Images as requested

File: examples/layouts/ElevationView.vue [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<template>
+  <v-container grid-list-xl text-xs-center>
+    <v-spacer></v-spacer>
```
Comment (johnleider) [inline examples/layouts/ElevationView.vue]: is this needed?

File: examples/layouts/ElevationView.vue [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    <v-layout row wrap>
+      <v-flex xs6 offset-md3 offset-xs1>
+        <v-card>
+          <v-slider v-model="selected" prepend-icon="fa-hand-paper-o" min="0" max="24" class="pl-3" thumb-label>
```
Comment (johnleider) [inline examples/layouts/ElevationView.vue]: format these props into new lines, > 2 should always be formatted on new lines

File: examples/layouts/ElevationView.vue [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        </v-card>
+      </v-flex>
+    </v-layout>
+  </v-container>
+</template>
+
+<script>
+export default {
+  name: 'ElevationView',
+  data() {
```
Comment (johnleider) [inline examples/layouts/ElevationView.vue]: space after **data**

File: examples/layouts/ImageView.vue [hunk 4]
```diff
@@ -0,0 +1,32 @@
+<template>
+  <v-container grid-list-xl text-xs-center>
+    <v-spacer></v-spacer>
+    <v-layout row wrap>
+      <v-flex xs6 offset-md3 offset-xs1>
+        <v-card>
+          <v-slider v-model="selected" prepend-icon="fa-arrows-h" min="0" max="24" class="pl-3" thumb-label>
```
Comment (johnleider) [inline examples/layouts/ImageView.vue]: format props

File: examples/layouts/ImageView.vue [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+          </v-slider>
+        </v-card>
+      </v-flex>
+      <v-spacer></v-spacer>
+      <v-flex xs6 offset-md3 offset-xs1>
+        <v-card v-bind:class="{ [`elevation-${selected}`]: true }">
+          <v-card-media height="300px" :src="`https://picsum.photos/300/400?image=586`">
```
Comment (johnleider) [inline examples/layouts/ImageView.vue]: does not need to be a bound prop

File: examples/layouts/TextView.vue [hunk 6]
```diff
@@ -0,0 +1,34 @@
+<template>
+  <v-container grid-list-xl text-xs-center>
+    <v-spacer></v-spacer>
+    <v-layout row wrap>
+      <v-flex xs6 offset-md3 offset-xs1>
+        <v-card>
+          <v-slider v-model="selected" prepend-icon="fa-hand-o-right" min="0" max="24" class="pl-3" thumb-label>
```
Comment (johnleider) [inline examples/layouts/TextView.vue]: format props

File: examples/layouts/TextView.vue [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+          </v-slider>
+        </v-card>
+      </v-flex>
+      <v-spacer></v-spacer>
+      <v-flex xs6 offset-md3 offset-xs1>
+        <v-card v-bind:class="{ [`elevation-${selected}`]: true }">
+          <v-card-text text-xs-center ma-0 class="blue-grey lighten-3 black--text">
```
Comment (johnleider) [inline examples/layouts/TextView.vue]: format props

File: examples/layouts/TextView.vue [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+              Phasellus feugiat arcu sapien, et iaculis ipsum elementum sit amet. Mauris cursus commodo interdum. Praesent ut risus eget metus luctus accumsan id ultrices nunc. Sed at orci sed massa consectetur dignissim a sit amet dui. Duis commodo vitae velit et
+              faucibus. {{ selected }}
+            </div>
+          </v-card-text>
+        </v-card>
+      </v-flex>
+    </v-layout>
+  </v-container>
+</template>
+<script>
```
Comment (johnleider) [inline examples/layouts/TextView.vue]: space between template and script

File: examples/layouts/ElevationView.vue (hunk 1 above)
Comment (Leehman) [reply examples/layouts/ElevationView.vue]: No fixed as requested.

File: examples/layouts/ElevationView.vue (hunk 3 above)
Comment (Leehman) [reply examples/layouts/ElevationView.vue]: Fixed as requested.

File: examples/layouts/ElevationView.vue (hunk 2 above)
Comment (Leehman) [reply examples/layouts/ElevationView.vue]: Reformatted as requested.

File: examples/layouts/ImageView.vue (hunk 4 above)
Comment (Leehman) [reply examples/layouts/ImageView.vue]: Corrected

File: examples/layouts/ImageView.vue (hunk 5 above)
Comment (Leehman) [reply examples/layouts/ImageView.vue]: Corrected.

File: examples/layouts/TextView.vue (hunk 6 above)
Comment (Leehman) [reply examples/layouts/TextView.vue]: Fixed as requested.

File: examples/layouts/TextView.vue (hunk 7 above)
Comment (Leehman) [reply examples/layouts/TextView.vue]: Props formatted.

File: examples/layouts/TextView.vue (hunk 8 above)
Comment (Leehman) [reply examples/layouts/TextView.vue]: Fixed.

**Comment by Leehman** [comment]
Note that the the original push was to examples/layout should have been to pages/layout.

File: package.json [hunk 9]
```diff
@@ -60,7 +60,7 @@
     "babel-preset-stage-2": "^6.24.1",
     "babel-register": "^6.26.0",
     "css-loader": "^0.28.7",
-    "eslint": "^4.15.0",
+    "eslint": "^4.16.0",
```
Comment (johnleider) [inline package.json]: revert this

File: pages/layout/ElevationView.vue [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+    </v-layout>
+  </v-container>
+</template>
+<script>
+export default {
+  name: "ElevationView",
+  data() {
+    return {
+      selected: 0
+    };
```
Comment (johnleider) [inline pages/layout/ElevationView.vue]: no semi colon

File: pages/layout/ImageView.vue [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+    </v-layout>
+  </v-container>
+</template>
+<script>
+export default {
+  name: "ImageView",
+  data() {
+    return {
+      selected: 0
+    };
```
Comment (johnleider) [inline pages/layout/ImageView.vue]: no semi colon

File: pages/layout/TextView.vue [hunk 12]
```diff
@@ -0,0 +1,42 @@
...
+    </v-layout>
+  </v-container>
+</template>
+<script>
+export default {
+  name: "TextView",
+  data() {
+    return {
+      selected: 0
+    };
```
Comment (johnleider) [inline pages/layout/TextView.vue]: no semi colon

File: package.json (hunk 9 above)
Comment (Leehman) [reply package.json]: Fixed.

File: pages/layout/ElevationView.vue (hunk 10 above)
Comment (Leehman) [reply pages/layout/ElevationView.vue]: Fixed.

File: pages/layout/ImageView.vue (hunk 11 above)
Comment (Leehman) [reply pages/layout/ImageView.vue]: Fixed.

File: pages/layout/TextView.vue (hunk 12 above)
Comment (Leehman) [reply pages/layout/TextView.vue]: Fixed pushing back up.

**Comment by Leehman** [comment]
Clean and available to review.

**Comment by Leehman** [comment]
i found elevation.js and placed it in lang\en\components. however the original place it used to be in pages\layout and it was .vue . it failed on the CI push to the repo. best guess i moved it to somewhere the CI didn't like. trying to resolve.

**Comment by Leehman** [comment]
snap 2018-02-14 at 10 58 54
snap 2018-02-13 at 11 23 46
snap 2018-02-14 at 10 12 03

**Comment by Leehman** [comment]
JL - i created a new directory 'elevations' in examples which has the 3 examples in it.

**Pull request closed without merging**